modules/cassandra/src/main/java/org/testcontainers/cassandra/CassandraContainer.java

```diff
@@ -0,0 +1,168 @@
+package org.testcontainers.cassandra;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import org.testcontainers.cassandra.delegate.CassandraDatabaseDelegate;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.ext.ScriptUtils;
+import org.testcontainers.ext.ScriptUtils.ScriptLoadException;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+import java.io.File;
+import java.net.InetSocketAddress;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Optional;
+
+/**
+ * Testcontainers implementation for Apache Cassandra.
+ * <p>
+ * Supported image: {@code cassandra}
+ * <p>
+ * Exposed ports: 9042
+ */
+public class CassandraContainer extends GenericContainer<CassandraContainer> {
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("cassandra");
+
+    private static final Integer CQL_PORT = 9042;
+
+    private static final String DEFAULT_LOCAL_DATACENTER = "datacenter1";
+
+    private static final String CONTAINER_CONFIG_LOCATION = "/etc/cassandra";
+
+    private static final String USERNAME = "cassandra";
+
+    private static final String PASSWORD = "cassandra";
+
+    private String configLocation;
+
+    private String initScriptPath;
+
```

Comment:
Let's use this one in tests

```suggestion
    public CassandraContainer(String dockerImageName) {
        this(DockerImageName.parse(dockerImageName);
    }
```